pandas/tests/io/json/test_readlines.py

```diff
@@ -54,7 +54,9 @@ def test_read_datetime(request, engine):
     if engine == "pyarrow":
         result = read_json(StringIO(json_line), engine=engine)
     else:
-        result = read_json(StringIO(json_line), engine=engine)
+        msg = "parsing datetimes with mixed time zones will raise a warning"
+        with tm.assert_produces_warning(FutureWarning, match=msg):
+            result = read_json(StringIO(json_line), engine=engine)
```

Comment:
How can a user address this warning from any IO method if they don't expose a way to set utc=True?

Reply:
ooh that's a good point, thanks! OK we need to address this first